src/fedora/32/helix/amd64/Dockerfile

```diff
@@ -1,11 +1,36 @@
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-32
 
+# Add MS package repo
+RUN dnf config-manager --add-repo https://packages.microsoft.com/fedora/32/prod/ && \
```

Comment:
Can't we cURL/wget the GPP public key?
Or are we constraint here by something or is it frown upon to not to hard code?

Reply:
No reason to not just curl this from a well-known public location if possible, lots of locations for it on the internet.

Reply:
we could. But I did not want to establish trust by depending on internet. If we want to do that, I think we should hardcore sha and verify that what ever we download is exactly what we expect. I would be probably ok with gpgp registry but I did not find our keys there.